quickinstall

```diff
@@ -1,249 +1,45 @@
-#!/bin/bash
-# ckb auto build/install script
+#!/usr/bin/env bash
 
-# Temporary files for program output
-TMPFILE=$(mktemp /tmp/ckb.XXXXXXXX)
-function newtmp {
-    rm -f "$TMPFILE"
-    TMPFILE=$(mktemp /tmp/ckb.XXXXXXXX)
-}
+export LC_ALL=C
 
-# Check for failure, quit if unsuccessful
 function checkfail {
     if [[ $1 -ne 0 ]]; then
-        cat "$TMPFILE"
-        finish "$1"
+        echo "An error occurred,"
+        echo "press enter to exit."
+        read -rs dummy
+        exit "$1"
     fi
 }
 
-# Exit function
 function finish {
-    rm -f "$TMPFILE"
-    echo ""
-    echo "Press enter to exit."
-    read -rs dummy
-    exit "$1"
+    echo "Done."
+    echo "Thank you for installing ckb-next!"
+    exit 0
 }
 
-# Directories
-cd "$(dirname "$0")" || exit
-SRCDIR="src"
-BINDIR="bin"
-[[ "$OSTYPE" == "darwin"* ]] && BINDIR="ckb.app"
-# This script respects the PREFIX variable on Linux (always /Applications on OSX)
-[[ $PREFIX == "" ]] && PREFIX="/usr"
-[[ "$OSTYPE" == "darwin"* ]] && PREFIX="/Applications"
-
-tab=$'\t'
-bad_chars=" |'|$tab" # characters needed to be escaped
-if [[ "$(pwd)" =~ $bad_chars ]]; then
-    echo "Error: you must compile ckb in a directory with no spaces/single quotes/tabs."
-    echo "Move or rename the folder and try again."
-    finish 1
+# Check if we're in the right directory
+if [[ ! -f "src/daemon/usb.c" ]]; then
+    echo "Please run this script inside the ckb-next source directory."
+    exit
 fi
 
-# Install command line tools if missing
-if [[ "$OSTYPE" == "darwin"* ]]; then
-    if xcode-select --install 2>&1 | grep -q "installed"; then
-        echo "Command line tools are already installed."
-    else
-        # check if user installed them after the gui prompt and prompt him once again
-        if xcode-select --install 2>&1 | grep -q "installed"; then
-            echo "Command line tools are installed."
-        else
-            echo "Please install command line tools and rerun the script."
-            finish 1
-        fi
-    fi
-    # Check for Xcode presence
-    if xcodebuild -version 2>&1 | grep "error"; then
-        echo "Please install Xcode and rerun the script."
-        finish 1
-    fi
-fi
+git clean -xdf
```

Comment:
Super small concern: What if the user downloaded the source code via zip?

I admit it's a super obscure edge-case, but this means we should have `git` as a dependency for this project.

Reply:
I don't think calling git clean in quickinstall is a good idea.
People tend to make changes we tell them to do, and then call quickinstall, as they don't know how else to compile it, before we even have a chance to say anything.